modules/Authentication.php

```diff
@@ -1,11 +1,17 @@
 <?php
 	class Authentication {
-		private $user;
+		public $user;
```

Comment:
You should make this private and use getters to get this variable in any other file.
Think of this from a software design perspective. You would never want any other script to modify the user data in the authentication file.